src/main/resources/application-sentry.yml

```
@@ -0,0 +1,7 @@
+sentry:
+  dsn: ${SENTRY_DSN}
+  debug: true
+  server-name: ${SENTRY_SERVER_NAME}
+  send-default-pii: true
+  exception-resolver-order: 1
+  max-breadcrumbs: 30
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)